tutorials/Tutorial5_Evaluation.py

```diff
@@ -170,34 +192,53 @@ def tutorial5_evaluation():
     metrics = advanced_eval_result.calculate_metrics()
     print(metrics["Reader"]["sas"])
 
-    ## Isolated Evaluation Mode to Understand Upper Bounds of the Reader's Performance
-    # The isolated node evaluation uses labels as input to the reader node instead of the output of the preceeding retriever node.
-    # Thereby, we can additionally calculate the upper bounds of the evaluation metrics of the reader.
+    ## Isolated Evaluation Mode
+    # The isolated node evaluation uses labels as input to the Reader node instead of the output of the preceeding retriever node.
+    # Thereby, we can additionally calculate the upper bounds of the evaluation metrics of the Reader.
+    # Note that even with isolated evaluation enabled, integrated evaluation will still be running.
+    eval_labels = [label for label in eval_labels if not label.no_answer] # filter out no_answer cases
```

Comment:
Users might be surprised that we need to do this in isolated node eval. I would prefer moving this line of code to where we define `eval_labels`. Let's explain you could do the evaluation with or without no_answer labels and what the difference is.